frontend/viewer/src/lib/components/ui/sonner/sonner.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+  import {mode} from 'mode-watcher';
+  import {Toaster as Sonner, type ToasterProps as SonnerProps} from 'svelte-sonner';
+
+  let restProps: SonnerProps = $props();
+</script>
+
+<Sonner
```

Comment:
It looks like we can customize the icons used, I'm not sure where it's icons are currently coming from
https://github.com/wobsoriano/svelte-sonner?tab=readme-ov-file#changing-icon

Reply:
Nice. Yeah, I guess it would make sense to use consistent icons. 👍